day-5.lisp

```diff
@@ -0,0 +1,51 @@
+(defpackage aoc-day-5
+  (:use :cl)
+  (:export :answers))
+(in-package :aoc-day-5)
+
+(defun is-nice-old (word)
+  (if (or (search "ab" word)
+          (search "cd" word)
+          (search "pq" word)
+          (search "xy" word))
+      nil
+      (let ((prev-char #\Nul)
+            (num-vowels 0)
```

Comment:
Why initialize this with a default value but `not has-repeated-char` ?

Reply:
That's because I want `num-vowels` to be an integer whereas `has-repeated-char` to be a boolean. In lisp, booleans can be `t` (true) or `nil` (false). In `let`, uninitialized variables are set to `nil`.

On a related note, in lisp, `nil` is synonymous with an empty list as well where applicable.